infra/modules/azure_function_app_exposed/variables.tf

```diff
@@ -0,0 +1,123 @@
+variable "tags" {
+  type        = map(any)
+  description = "Resources tags"
+}
+
+variable "environment" {
+  type = object({
+    prefix          = string
+    env_short       = string
+    location        = string
+    domain          = optional(string)
+    app_name        = string
+    instance_number = string
+  })
+
+  validation {
+    condition     = length("${var.environment.prefix}${var.environment.env_short}reg${var.environment.domain == null ? "" : replace(var.environment.domain, "-", "")}${var.environment.app_name}stfn${var.environment.instance_number}") <= 24
```

Comment:
isn't this a different string pattern than the one used to create the storage account here

https://github.com/pagopa/dx/pull/103/files#diff-e9cd22b1ffa1bbcc682f793bf8a6c49022270e089e609b4dbf7f33e0ee8755bbR29 ?

Reply:
`prefix` and `env_short` don't contain any `-`, while `project` does, that's why in `locals.tf` there is a `replace` at the beginning.
Anyway, we will move both of them in `azure_naming_convention` module making everything more consistent through all modules